tsconfig.json

```diff
@@ -64,5 +64,26 @@
     "babel.config.js",
     "metro.config.js",
     "jest.config.js"
+  ],
+  "plugins": [
+    {
+      "name": "typescript-styled-plugin",
+      "validate": false,
+      "lint": {
+        "validProperties": [
```

Comment:
This _should_ allow the following CSS properties to be used with `styled-components`, but it might not work properly with VSCode anyways:
https://github.com/microsoft/typescript-styled-plugin/issues/58